fdbsyncd/fdbsyncd.cpp

```diff
@@ -67,14 +91,39 @@ int main(int argc, char **argv)
             {
                 s.select(&temps);
 
-                if(temps == (Selectable *)sync.getFdbStateTable())
+                if (temps == (Selectable *)sync.getFdbStateTable())
                 {
                     sync.processStateFdb();
                 }
                 else if (temps == (Selectable *)sync.getCfgEvpnNvoTable())
                 {
                     sync.processCfgEvpnNvo();
                 }
+                else if (temps == &replayCheckTimer)
+                {
+                    if (sync.getFdbStateTable()->empty() && sync.getCfgEvpnNvoTable()->empty())
+                    {
+                        sync.getRestartAssist()->appDataReplayed();
+                        SWSS_LOG_NOTICE("FDB Replay Complete");
+                        s.removeSelectable(&replayCheckTimer);
+
+                        /* Obtain warm-restart timer defined for routing application */

```

Comment:
Add a comments here for:
 - fdb dependencies on routing application
 - We should have TBD comment for addressing optimization of EOIU etc later. IMO, checking the bgp reconciliation state is a better way to handle the dependency.  If we really have to do it next, let's track it with an issue, and add it to the code comment.

Reply:
Sure will add the comment

Reply:
Submitted issue https://github.com/Azure/sonic-swss/issues/1657 to track the eoiu implementation for EVPN AF